kratos/python_scripts/KratosUnittest.py

```diff
@@ -25,10 +25,15 @@ def loadTestsFromTestCases(self, testCaseClasses):
         return allTests
 
 
+test_timing_results = {}
+
 class TestCase(TestCase):
 
     def run(self, result=None):
-        super(TestCase,self).run(result)
+        start_time = time()
```

Comment:
Would `perf_counter` be more suitable? Not sure what it does if MPI is involved though. https://stackoverflow.com/questions/25785243/understanding-time-perf-counter-and-time-process-time

Reply:
I tired it (in fact was initial suggestion from @roigcarlo) but the output is, well, profiling output
I.e. according to this, the `Run` method of he AnalyisStage is the slowest in the tests

In a nutshell: not really suitable

Reply:
For me it simply gives a float:
```
>>> import time
>>> a = time.perf_counter()
>>> print(time.perf_counter() - a)
14.392021126986947
```

Reply:
I think the main difference is that perf_counter would take into account the loop time for the MPI collectives, which would be slightly different if the transfers are synch and large, but other than that I see no problem in that.

> I tired it (in fact was initial suggestion from @roigcarlo)

Did I? I think  we spoke about `cProfile`, but not `perf_counter` :S

Reply:
sorry I confused `cProfile` with `perf_counter` :D

@armingeiser the only difference is that `perf_counter` is more accurate no?
=> makes no difference in this case since I output only the first 2 digits, thought it would be sufficient for this purpose